components/brave_rewards/browser/database/database_unblinded_token.cc

```diff
@@ -154,4 +161,21 @@ bool DatabaseUnblindedToken::DeleteRecord(
   return statement.Run();
 }
 
+bool DatabaseUnblindedToken::DeleteRecordsForPromotion(
```

Comment:
nit: if function is static, we normally add `// static` above function

Reply:
Done